src/command_modules/azure-cli-acr/README.rst

```diff
@@ -0,0 +1,205 @@
+Microsoft Azure CLI 'acr' Command Module
+==================================
+
+This package has [not] been tested [much] with Python 2.7, 3.4 and 3.5.
```

Comment:
>[not] been tested [much] [](start = 17, length = 24)

We really should remove this from the example package (well, and here too). @derekbekoe do you agree?